src/components/Calculator.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+// import './calc.css';
+
+class calculator extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {value: ''};
+    }
+
+    render() {
+        return (
+            <div id="CON">
+                
+                <section id="buttons">
+                    <div id="screen">0</div>
+                    <button>AC</button>
+                    <button>+/-</button>
+                    <button>%</button>
+                    <button className="last-row">/</button>
+                    
+                    
+                    <button>7</button>
```

Comment:
- [ ] OPTIONAL
Kindly ensure not to add leave empty lines in your code which are not serving any purpose, so as to keep your code well organized.